bindings/python/edlib.pyx

```diff
@@ -152,3 +154,91 @@ def align(query, target, mode="NW", task="distance", k=-1, additionalEqualities=
     cedlib.edlibFreeAlignResult(cresult)
 
     return result
+
+
+
+## function to output NICE alignments
+## only 'task=="path"' works
+
+def cigarToNiceAlignment(alignResult, query, target, gapSymbol="-"):
+    """ Output alignments from align() in NICE format
+    @param {dictionary} alignResult, output of align() 
+            Dictionary with following fields:
+            {int} editDistance  -1 if it is larger than k.
+            {int} alphabetLength
+            {[(int, int)]} locations  List of locations, in format [(start, end)].
+            {string} cigar  Cigar is a standard format for alignment path.
+                Here we are using extended cigar format, which uses following symbols:
+                Match: '=', Insertion to target: 'I', Deletion from target: 'D', Mismatch: 'X'.
+                e.g. cigar of "5=1X1=1I" means "5 matches, 1 mismatch, 1 match, 1 insertion (to target)".
+    @param {str or bytes or iterable of hashable objects} query, combined with target must have no more
```

Comment:
Here, don't describe query like this. Just say it should be query that was used to obtain alignResult. Same goes for target.

Reply:
Agreed. I also think it needs to be a string, right?

Reply:
I'll let you check the updates, @Martinsos :)